app/views/teams/_form.html.slim

```diff
@@ -69,8 +69,8 @@
               = s.link_to_remove 'Remove', class: 'btn btn-default'
       .form-group.form-btn-group
         = f.link_to_add 'Add another source', :sources, class: 'btn btn-primary col-sm-offset-3'
-  -if can?(:update_conference_preferences, @team)
-  = render 'conference_preference_fields', f: f
+  - if can?(:update_conference_preferences, @team) && @team.accepted?
```

Comment:
Yes, something along the lines of this. For admin users, it should not make a difference whether or not the team is accepted - I would in fact generally remove the admins here in this ability, as orga has their own team form (there you can simply enable all). And I would probably do the check for `accepted?` first.